init.sh

```diff
@@ -39,6 +42,23 @@ echo "------"
 echo resolver $(awk 'BEGIN{ORS=" "} /nameserver/{print $2}' /etc/resolv.conf | sed "s/ $/;/g") > /etc/api-gateway/conf.d/includes/resolvers.conf
 echo "   ...  with dns $(cat /etc/api-gateway/conf.d/includes/resolvers.conf)"
 
+sync_cmd="echo checking for changes ..."
+if [[ -n "${remote_config}" ]]; then
+    echo "   ... using a remote config from: ${remote_config}"
+    if [[ ${remote_config} == s3://* ]]; then
```

Comment:
It doesn't work exactly like that, but it works like `"${remote_config}" == s3://*`. Maybe a RegEx would make it clearer ? `"${remote_config}" =~ ^s3://.+`